amplify/src/main/java/com/github/stkent/amplify/tracking/IntegratedEvent.java

```diff
@@ -20,12 +20,12 @@
 import android.support.annotation.NonNull;
 
 import com.github.stkent.amplify.tracking.interfaces.IEvent;
+import com.github.stkent.amplify.tracking.interfaces.ILogger;
 
 public enum IntegratedEvent implements IEvent {
 
     APP_CRASHED,
     APP_INSTALLED,
-    APP_UPDATED,
```

Comment:
I didn't mean to take that out, it's not currently used so I took it out but realized someone else might want to use it.